src/smallMultiples.js

```diff
@@ -68,7 +68,7 @@ export default function smallMultiples(id, chart) {
         rangePolygon(this);
 
         // axis tweaks
-        var units = this.current_data[0].values.raw[0][chart.config.unit_col];
+        var units = this.raw_data[0][chart.config.unit_col];
```

Comment:
might need something like `raw_data.filter(d=>give me the current measure)[0][chart.config.unit_col]`